include/z64.h

```diff
@@ -1319,8 +1338,8 @@ typedef struct {
     /* 0x5C */ u8  (*switchFromRoom)[51]; // room to come from
     /* 0x60 */ u8  (*switchFromFloor)[51]; // floor to come from
     /* 0x64 */ u8  (*switchToRoom)[51]; // room to go to
-    /* 0x68 */ UNK_PTR unk_68;
-    /* 0x6C */ UNK_PTR unk_6C;
+    /* 0x68 */ u8  (*floorNames)[8];
```

Comment:
I think these would be better as "IDs" rather than "names", with the enum and array names adjusted as well of course. Also the rest of the struct uses singular names so I suggest `floorId`.